products/sle/main.pm

```diff
@@ -317,10 +317,12 @@ if (get_var('ENABLE_ALL_SCC_MODULES') && !get_var('SCC_MODULES')) {
     if (sle_version_at_least('15')) {
         # let's start with what corresponds to server
         set_var('SCC_ADDONS', 'base,script,desktop,serverapp,phub,legacy,sdk,ha,wsm');
+        set_var('PATTERNS', 'default,asmm,pcm');
     }
     else {
         if (check_var('ARCH', 'aarch64')) {
             set_var('SCC_ADDONS', 'pcm,tcm');
+            set_vat('PATTERNS','default, pcm');
```

Comment:
yeah, you did not really test that ;-)

Reply:
Yeah aarch64 is always a bit difficult :P